xml/obs_best_practice_kiwi_editor.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+
+<section id="kiwi_editor_how_to">
+ <title>KIWI Editor</title>
+ <para>
+   You can edit the KIWI file associated to your project. If you are running your own instance of OBS be sure you have the kiwi_image_editor feature enabled in your config/feature.yml file.
```

Comment:
I think it worth to speak about the actual state of the editor, at least that we right now only operate on the Repositories section of the file.